central/complianceoperator/v2/report/manager/manager_impl.go

```diff
@@ -0,0 +1,24 @@
+package manager
+
+import (
+	"context"
+
+	scanSettingBindingsDS "github.com/stackrox/rox/central/complianceoperator/v2/scansettingbindings/datastore"
+	v2 "github.com/stackrox/rox/generated/api/v2"
+)
+
+type managerImpl struct {
+	datastore scanSettingBindingsDS.DataStore
```

Comment:
this will need a channel to accept report requests, and also possibly an in memory hashmap of scan config id -> report request to determine if a request for this scan config id is already submitted/being processed

We also will need an in mem report request struct